Makefile

```diff
@@ -316,16 +316,16 @@ create-cluster: $(KUSTOMIZE) $(ENVSUBST) ## Create a development Kubernetes clus
 	mkdir ~/.cluster-api
 	chmod +x $(CLUSTERCTL)
 	# Create clusterctl.yaml to use local OpenStack provider
-	mkdir -p ./out/infrastructure-openstack/v0.3.1
+	mkdir -p ./out/infrastructure-openstack/pr
```

Comment:
what does pr stand for? I think the reason I added it might have been because clusterctl expected it in this folder. But not sure

Reply:
yes, my test showed that ,but I think v0.3.1 does not make sense now... at least need move to 0.3.2 or 0.3.3
checking the reason and dependency you mentioned above now..

Reply:
Yup. Let's see what we need here :). I don't really know anymore where the dependency came from if there is one

Reply:
FYI: CAPA uses next minor version: v0.7.0
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/clusterctl-settings.json#L5

Reply:
Error: failed to get provider components for the "openstack" provider: failed to get repository client for the InfrastructureProvider with name openstack: error creating the local filesystem repository client: invalid version: "pr". Version must obey the syntax and semantics of the "Semantic Versioning" specification (http://semver.org/) and path format {basepath}/{provider-name}/{version}/{components.yaml}

this is the issue you mentioned @sbueringer 
so we need switch to 0.3.3 and update it whenever we have a new version ...